docs/cmd/kn_service_create.md

```diff
@@ -62,10 +62,11 @@ kn service create NAME --image IMAGE [flags]
   -n, --namespace string          Specify the namespace to operate in.
       --no-lock-to-digest         do not keep the running image for the service constant when not explicitly specifying the image. (--no-lock-to-digest pulls the image tag afresh with each new revision)
   -p, --port int32                The port where application listens on.
+      --pull-secrets string       Image pull secrets to set. Empty image pull secrets will the clear the pull secrets.
       --requests-cpu string       The requested CPU (e.g., 250m).
       --requests-memory string    The requested memory (e.g., 64Mi).
       --revision-name string      The revision name to set. Must start with the service name and a dash as a prefix. Empty revision name will result in the server generating a name for the revision. Accepts golang templates, allowing {{.Service}} for the service name, {{.Generation}} for the generation, and {{.Random [n]}} for n random consonants. (default "{{.Service}}-{{.Random 5}}-{{.Generation}}")
-      --service-account string    Service account name to set. Empty service account name will result to clear the service account.
+      --service-account string    Service account name to set. Empty service account name will clear the service account.
```

Comment:
```suggestion
      --service-account string    Service account name to set. An empty argument clears the service account. The referenced service account must exist in the service's namespace.
```

Reply:
well, actually I suggest the change on in the edit_flags.go file. But youi know whay I mean ;-)